README.md

```diff
@@ -71,6 +71,14 @@ And this is what gets rendered (plus the CSS and Javascript you've specified):
 
 ## Release notes
 
+🚨📢 **Version 0.100**
+- BREAKING CHANGE:
+    - `django_components.safer_staticfiles` app was removed. It is no longer needed.
+    - Installation change: Instead of defining component directories in `STATICFILES_DIRS`, set them to `COMPONENTS.dirs`.
+    - Installation change: You now must define `STATICFILES_FINDERS`
+    - [See here how to migrate your settings.py](https://github.com/EmilStenstrom/django-components/pull/652/files#diff-59eb658f2c5573a64306ac0adc96405344e029fd2df51573555c82e44b4d7062)
```

Comment:
I think this is too hard to follow. Don't think everyone is comfortable following git diffs around I'm afraid. Maybe we need a new page to link to?